SPECS/golang/golang-1.17.spec

```diff
@@ -22,6 +22,8 @@ URL:            https://golang.org
 Source0:        https://golang.org/dl/go%{version}.src.tar.gz
 Source1:        https://dl.google.com/go/go1.4-bootstrap-20171003.tar.gz
 Patch0:         go14_bootstrap_aarch64.patch
+# CVE-2022-41717 is fixed in 1.18.9, but has not been back ported to 1.17.x
+Patch1:         CVE-2022-41717.patch
```

Comment:
I think you forgot to bump the `Release` tag and add a changelog entry.

Reply:
Sorry, realized I had another bug and am still working on the commit for this, will mark as draft for now.